README.md

```diff
@@ -1,492 +1,271 @@
-# PactNet
+<span align="center">
 
-[![Build status](https://ci.appveyor.com/api/projects/status/5h4t9oerlhqcnwm8/branch/master?svg=true)](https://ci.appveyor.com/project/PactFoundation/pact-net/branch/master)  
-A .NET implementation of the Ruby consumer driven contract library, [Pact](https://github.com/realestate-com-au/pact).  
-Pact is based on the specification found at https://github.com/pact-foundation/pact-specification.  
+![logo](https://user-images.githubusercontent.com/53900/121775784-0191d200-cbcd-11eb-83dd-adc001b94519.png)
 
-PactNet primarily provides a fluent .NET DSL for describing HTTP requests that will be made to a service provider and the HTTP responses the consumer expects back to function correctly.  
-In documenting the consumer interactions, we can replay them on the provider and ensure the provider responds as expected. This basically gives us complete test symmetry and removes the basic need for integrated tests.  
-PactNet also has the ability to support other mock providers should we see fit.
+# Pact Net
 
-PactNet is Version 2.0 compliant, and we now use the [Ruby standalone engine](https://github.com/pact-foundation/pact-ruby-standalone) as we move towards a common core approach. To enable Version 2.0 support,  make sure you supply a `PactConfig` object with `SpecificationVersion = "2.0.0"` when creating the `PactBuilder`.  
+[![Build status](https://ci.appveyor.com/api/projects/status/5h4t9oerlhqcnwm8/branch/master?svg=true)](https://ci.appveyor.com/project/PactFoundation/pact-net/branch/master)

```

Comment:
Oh forgot to mention - the CI is now done in GitHub Actions instead of AppVeyor. We should probably update the badge. Ci is here: https://github.com/pact-foundation/pact-net/actions/workflows/ci.yml